llvm/lib/Target/RISCV/RISCVSubtarget.h

```diff
@@ -99,6 +99,7 @@ class RISCVSubtarget : public RISCVGenSubtargetInfo {
   bool HasExtXCoreVMac = false;
   bool HasExtXCoreVAlu = false;
   bool HasExtXCoreVMem = false;
+  bool HasExtXCoreVSIMD = false;
```

Comment:
Consider also changing `hasExtXCoreVSIMD` to `hasExtXcvsimd` here. This applies to all instances of this variable but I will not comment on any more.

Reply:
I see some changes have been made. In general use `cvsimd` rather than `corevsimd` throughout for consistency.

Reply:
The name of this had been changed along with all other `corevsimd`-like names, the comment points to an old version of it.

Reply:
Checked by @CharKeaney and loosk good.